packages/ses/scripts/bundle.js

```diff
@@ -0,0 +1,37 @@
+import '../index.js';
+import fs from 'fs';
```

Comment:
We should adopt some convention that we can check, for making sources of static authority stand out in our source files. Just noting. No change suggested at this time. Attn @dckc

Reply:
This should also be a script that we run with Endo instead of Node, and it should be counted a design goal for Endo to accommodate injection of an attenuated FS and a decent DX for granting that capability to the script. For example, there might be some reasonable default filesystem attenuation for any Endo script run in the context of developer tools for a package, or at least a convenient piece of boilerplate we can add to package.json to grant that limited authority.

Reply:
"DX"?